.github/workflows/test.yml

```diff
@@ -0,0 +1,51 @@
+name: Tests
+on:
+  push: # Run on all branches
+  pull_request:
+    branches: [develop, main]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.12"]
```

Comment:
I missed this! Let's rely on the .python-version file in the repo until the point where we ever need to test more than one version of python.  The setup-python action uses this automatically.